robottelo/ui/base.py

```diff
@@ -516,3 +533,26 @@ def perform_action_chain_move(self, locator):
                 .format(type(self).__name__, locator)
             )
         ActionChains(self.browser).move_to_element(element).perform()
+
+    def assign_value(self, locator, value):
```

Comment:
Goal:
We created that method to be able to interact with element without knowing its type in advance. That is very useful when you have algorithm that go through entire page and do corresponding actions. For example, click if you have button or fill form fields with necessary values.

That is way out from too straightforward approaches used in our code before and lead to minimization of used code and effrots